etc/profile-m-z/signal-desktop.profile

```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
 netfilter
 nodvd
 nogroups
+noroot
 notv
 nou2f
+novideo
```

Comment:
IIRC signal has a video-chat. IDK if it is also supported by the desktop-version.

Reply:
desktop version don't have video-chat. Only Android/iOS version